WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -123,13 +122,6 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
 
-    if (self.previousOrientation != UIInterfaceOrientationUnknown) {
-        if (UIInterfaceOrientationIsPortrait(self.previousOrientation) != UIInterfaceOrientationIsPortrait(self.interfaceOrientation)) {
-            [self.tableViewHandler refreshCachedRowHeightsForWidth:CGRectGetWidth(self.view.frame)];
```

Comment:
@jleandroperez I reverted those changes @aerych will handle this in a separate PR.